src/main/java/ua/com/cinema/dao/AbstractDao.java

```diff
@@ -17,14 +17,10 @@ protected AbstractDao(SessionFactory sessionFactory, Class<T> clazz) {
         this.clazz = clazz;
     }
 
-    public SessionFactory getSessionFactory() {
+    protected SessionFactory getSessionFactory() {
```

Comment:
You can make your field protected and remove this getter

Reply:
😢